--- MultiWii.old/Serial.ino	2013-03-15 09:43:59.831324921 -0400
+++ MultiWii.new/Serial.ino	2013-03-16 05:32:19.280242687 -0400
@@ -72,6 +72,8 @@
 #define MSP_SELECT_SETTING       210   //in message          Select Setting Number (0-2)
 #define MSP_SET_HEAD             211   //in message          define a new heading hold direction
 
+#define MSP_USBLINKER            239   //in message          Select USB Linker mode
+
 #define MSP_BIND                 240   //in message          no param
 
 #define MSP_EEPROM_WRITE         250   //in message          no param
@@ -525,6 +527,9 @@
      }
      break;
    #endif
+   case MSP_USBLINKER:
+     { void AUL_loop(uint8_t); AUL_loop(CURRENTPORT); }
+     break;
    default:  // we do not know how to handle the (valid) message, indicate error MSP $M!
      headSerialError(0);
      break;
@@ -868,3 +873,5 @@
 #else
 void debugmsg_append_str(const char *str) {};
 #endif
+
+#include "ArduinoUSBLinker.h"
