--- Serial.ino.orig	2012-11-04 18:01:38.264449135 -0500
+++ Serial.ino	2012-11-05 14:12:28.736711865 -0500
@@ -68,6 +68,7 @@
 #define MSP_RESET_CONF           208   //in message          no param
 #define MSP_SET_WP               209   //in message          sets a given WP (WP#,lat, lon, alt, flags)
 #define MSP_SELECT_SETTING       210   //in message          Select Setting Number (0-2)
+#define MSP_SELECT_USBLINKER     211   //in message          Select USB Linker mode
 
 #define MSP_SPEK_BIND            240   //in message          no param
 
@@ -257,6 +258,9 @@
      headSerialReply(0);
      break;
 #endif
+   case MSP_SELECT_USBLINKER:
+     { void AUL_loop(uint8_t); AUL_loop(CURRENTPORT); }
+     break;
    case MSP_IDENT:
      headSerialReply(7);
      serialize8(VERSION);   // multiwii version
@@ -812,3 +816,5 @@
 #else
 void debugmsg_append_str(const char *str) {};
 #endif
+
+#include "ArduinoUSBLinker.h"
